tools/test-proxy/tests.yml

```diff
@@ -216,4 +220,59 @@ stages:
           workingDirectory: $(CLONE_LOCATION)/sdk/tables/azure-data-tables/
           env:
             PROXY_URL: "http://localhost:5000"
-            PROXY_MANUAL_START: true
\ No newline at end of file
+            PROXY_MANUAL_START: true
+
+    - job: Test_Java_Core
+
+      displayName: Run Java Core Local Tests
+
+      strategy:
+        matrix:
+          Windows:
+            Pool: 'azsdk-pool-mms-win-2022-general'
+            OS: 'Windows'
+          Linux:
+            Pool: azsdk-pool-mms-ubuntu-2204-general
+            OS: 'Linux'
+          Mac:
+            Pool: 'Azure Pipelines'
+            OS: 'Mac'
+
+      pool:
+        name: $(Pool)
+
+      variables:
+        REPO: "Azure/azure-sdk-for-java"
+        CLONE_LOCATION: "$(Agent.BuildDirectory)/java_repo"
+
+      steps:
+
+        - template: /eng/pipelines/templates/steps/install-dotnet.yml
+
+        - template: /eng/pipelines/templates/steps/use-python-version.yml
+          parameters:
+            versionSpec: '3.9'
+
+        - pwsh: |
+            git clone https://github.com/$(REPO) $(CLONE_LOCATION) --depth 1
+          displayName: Clone Repo
+
+        - pwsh: |
```

Comment:
You may consider skipping more stuff to make this faster:

```
-Dcodesnippet.skip -Drevapi.skip -Dspotbugs.skip -DskipTests -Djacoco.skip -DskipTestCompile
```

Of course for the actual `azure-core-test` don't skiptests or test compile.

Reply:
👍 I'll do so. Thanks Bill!